src/libm-tester/CMakeLists.txt

```diff
@@ -85,6 +85,20 @@ macro(test_extension SIMD)
     add_test_iut(${TARGET_IUT${SIMD}})
     list(APPEND IUT_LIST ${TARGET_IUT${SIMD}})
 
+    string(CONCAT IUTYNAME "iuty" ${LCSIMD})
```

Comment:
Please add a comment in the source explaining the difference between iut and iuty